packages/java/src/browser/java-commands.ts

```diff
@@ -109,10 +90,9 @@ export class JavaCommandContribution implements CommandContribution, MenuContrib
     }
 
     registerKeybindings(keybindings: KeybindingRegistry): void {
-        this.keybindingContextRegistry.registerContext(this.editorContext);
         keybindings.registerKeybinding({
             command: JAVA_ORGANIZE_IMPORTS.id,
-            context: CONTEXT_ID,
+            context: JavaKeybindingContexts.javaEditorActive,
```

Comment:
why is this registered for active editor and not focussed editor, only?

Reply:
fixed